packages/block-editor/src/components/spacing-sizes-control/spacing-input-control.js

```diff
@@ -227,7 +229,16 @@ export default function SpacingInputControl( {
 					<RangeControl
 						value={ customRangeValue }
 						min={ 0 }
-						max={ 300 }
+						max={
+							maxCustomValues[ selectedUnit ]
+								? maxCustomValues[ selectedUnit ]
+								: 10
```

Comment:
Could possibly change this to `maxCustomValues[ selectedUnit ] ?? 10`, but I also wonder why `em` and `rem` are excluded from `maxCustomValues`?